src/Config.php

```diff
@@ -1601,6 +1603,10 @@ public static function getAllConfigData()
         }
 
         include $configFile;
+        if (false === isset($phpCodeSnifferConfig)) {
+            $phpCodeSnifferConfig = null;
```

Comment:
Unless the config file - as include - unsets the variable. Had it first like you write, then moved it before the access, that's all.